kubejs/server_scripts/gamerules/default.js

```diff
@@ -0,0 +1,14 @@
+ServerEvents.loaded((event) => {
+    let gamerules = [
+        { rule: 'doFireTick', value: 'false' },
```

Comment:
I'm on the fence about this one, why change it?

Reply:
we always turn it off on our servers anyway, and having it on is just annoying. You find a nice forest to live in, AFK for a bit, and a lightning storm wipes it out with fire spread.

Or you build a nice wooden cabin and try to add a fire place, only for the whole place to go up in smoke.

Seemed like people appreciated it being off for e9e, so I thought I'd carry on with it.

Reply:
Sounds good 👍